libs/spark/src/Unstable_SectionMessage/Unstable_SectionMessage.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+import Unstable_SectionMessage from './Unstable_SectionMessage';
+
+describe('Unstable_SectionMessage', () => {
+  it('Can render without ThemeProvider', () => {
+    const { baseElement } = render(
+      <Unstable_SectionMessage title="Test">Testy Test</Unstable_SectionMessage>
```

Comment:
```suggestion
      <Unstable_SectionMessage title="Title">Message.</Unstable_SectionMessage>
```
sorry to spoil the fun